### ISSUE TITLE
Incorporate serializer into GitHub CI

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#905]
@dylan-sa observed that if someone accepts a suggestion in a PR review, it gets committed without going through the serializer. We need to include it in the CI in order to flag syntax errors in the ontology. Errors are not likely in the case of simple annotation changes, but if there are changes to axioms this could be a problem. 

In fact, we should be running a reasoner in the CI as well in order to catch logic errors.

Maybe there's a better way to do this. This approach of course doesn't prevent syntax or logic errors from being committed and pushed, but only flags them. Not everyone even looks at the build report.

**Comment by Jamie-SA** [comment]
Yup, I was already thinking about adding this so I've assigned it to me.
